client/src/components/Sponsors/Sponsors.js

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './Sponsors.css';
-import PartnersSection from '../Home/PartnersSection/PartnersSection';
-
-function Sponsors() {
-  return (
-    <div>
-      <PartnersSection />
-    </div>
-  );
-}
-
-export default Sponsors;
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Sponsors.css';
+// import PartnersSection from '../Home/PartnersSection/PartnersSection';

```

Comment:
So, are we going to have a new component for the Sponsors page?

Reply:
We will need to design a new sponsors page because if we pass the same ```<PartnersSection>``` refrence onto the sponsors component it will render the component present on the home page itself and since the component in home page includes the link to the sponsors page on clicking it will again redirect to the same sponsors page, which go on as a loop as many times we will click 😅.

Reply:
oops.. I think there must be a way around it. Why not redirect to a particular sponsor's website from the home page itself? @ankiiitraj.

Reply:
I think that will be fine as suggested by @EmperorYP7 🙌 .

Reply:
First of all, Should, we redirect to the Sponsor's website?
Do we require a page for sponsors? Yes as we may have to categories the sponsors!
Making a new component for this will be required I guess for more flexibility!
OR
Let me test something and get back on this!
create an issue for this!
and for now, we can merge this PR

Reply:
Alright. We can go with this now.